R/render-cached-plot.R

```diff
@@ -398,6 +400,8 @@ renderCachedPlot <- function(expr,
             func = isolatedFunc,
             width = width,
             height = height,
+#jy: default to title when NULL
+            alt = alt,
```

Comment:
Similar to `renderPlot()`, define `altWrapper` right after `fitDims` is defined on line 344.